Makefile

```diff
@@ -2,6 +2,9 @@
 
 # set the graph driver as the current graphdriver if not set
 DOCKER_GRAPHDRIVER := $(if $(DOCKER_GRAPHDRIVER),$(DOCKER_GRAPHDRIVER),$(shell docker info 2>&1 | grep "Storage Driver" | sed 's/.*: //'))
+export DOCKER_GRAPHDRIVER
+DOCKER_INCREMENTAL_BINARY := $(if $(DOCKER_INCREMENTAL_BINARY),$(DOCKER_INCREMENTAL_BINARY),1)
```

Comment:
It set's `DOCKER_INCREMENTAL_BINARY` to `1` if not already specified as a enviroment variable.
This allows you to do `make DOCKER_INCREMENTAL_BINARY=0 binary` to disable it for example.